Assets/MixedRealityToolkit.Services/InputSimulation/SimulatedHandPose.cs

```diff
@@ -442,6 +447,36 @@ public string GenerateInitializerCode()
             new Vector3(-0.06393f, 0.08967f, -0.03293f), // PinkyDistalJoint
             new Vector3(-0.06748f, 0.09752f, -0.02542f), // PinkyTip
         });
-    }
 
+        static private SimulatedHandPose HandOpenedSky = new SimulatedHandPose(new Vector3[]
```

Comment:
Out of curiosity: did you use the HandGestureRecorder tool to record this gesture?

Reply:
Hi Lukas,
At first, I didn't know the existence of this script, so I created my own MonoBehaviour.
When I found the HandGestureRecorder and tried to attach it to a gameobject, a the pop-up  shows : "The script is an editor script.".
So I created my own GestureRecorder